### DISCUSSION TITLE
How to use the method tablesample

### DISCUSSION THREAD

**Discussion by santigandolfo** [fastapi/sqlmodel#1501] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the SQLModel documentation, with the integrated search.
- [X] I already searched in Google "How to X in SQLModel" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to SQLModel but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to SQLModel but to [SQLAlchemy](https://github.com/sqlalchemy/sqlalchemy).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from sqlalchemy import func

selectable = people.tablesample(
            func.bernoulli(1),
            name='alias',
            seed=func.random())
stmt = select(selectable.c.people_id)
```

### Description

I'm trying to use the tablesample method defined in SQLAlchemy, but I can't seem to understand how to migrate it's structure from the sample code into SQLModel code.
For more context, I'm trying to get a random sample from a simple select query (with two wheres) and would prefere to do everything in the query instead of having to sample the resulting data (cause the table is really large)

### Operating System

macOS

### SQLModel Version

0.0.4

### Python Version

3.9.7

**Comment by YuriiMotov** [comment, marked as answer]
For further readers:
```py
from typing import Optional

from sqlalchemy import func, tablesample
from sqlmodel import Field, SQLModel, select

class People(SQLModel, table=True):
    people_id: Optional[int] = Field(default=None, primary_key=True)
    name: str

selectable = tablesample(People, func.bernoulli(1), name="alias", seed=func.random())

stmt = select(selectable.c.people_id)

print(stmt)
```

Output:
```
SELECT alias.people_id 
FROM people AS alias TABLESAMPLE bernoulli(:bernoulli_1) REPEATABLE (random())
```
